Source/WebCore/crypto/keys/CryptoKeyOKP.cpp

```diff
@@ -98,7 +98,7 @@ RefPtr<CryptoKeyOKP> CryptoKeyOKP::importJwk(CryptoAlgorithmIdentifier identifie
         if (keyData.crv != "Ed25519"_s)
             return nullptr;
         // FIXME: Do we have tests for these checks ?
```

Comment:
It seems this FIXME can be removed given that we have tests? Or if it doesn't apply to the following line maybe it should be more clearly scoped.

Reply:
Yes, we can remove the FIXME comment now.

Reply:
Done